symphony-bdk-core/src/main/java/com/symphony/bdk/core/HttpClient.java

```diff
@@ -0,0 +1,150 @@
+package com.symphony.bdk.core;
```

Comment:
This class should be located in the `http` module as well. It could be used outside of the `core`